README.md

```diff
@@ -86,6 +86,33 @@ Note: In this configuration, you must do some additional configuration:
 - If you wish your Plex Media Server to be accessible outside of your home network, you must manually setup port forwarding on your router to forward to the `ADVERTISE_IP` specified above.  By default you can forward port 32400, but if you choose to use a different external port, be sure you configure this in Plex Media Server's `Remote Access` settings.  With this type of docker networking, the Plex Media Server is essentially behind two routers and it cannot automatically setup port forwarding on its own.
 - (Plex Pass only) After the server has been set up, you should configure the `LAN Networks` preference to contain the network of your LAN.  This instructs the Plex Media Server to treat these IP addresses as part of your LAN when applying bandwidth controls.  The syntax is the same as the `ALLOWED_NETWORKS` below.  For example `192.168.1.0/24,172.16.0.0/16` will allow access to the entire `192.168.1.x` range and the `172.16.x.x` range.
 
+### ARM notes
```

Comment:
If build instructions are provided, I'd prefer it to not be ARM specific but rather generic to all platforms.

Reply:
The instructions are ARM-specific because there was not a working docker-compose template for ARM. It's just a couple extra details to help someone else know how to get a successful docker-compose working on ARM.

I considered making ARM-specific docker-compose templates, but it still required a note about running `docker build` locally anyway, so it made more sense (to me) to just add the template change to the notes as well, since it was such a small change.